index.bs

```diff
@@ -1382,18 +1382,28 @@ Votes</h3>
 	Procedures for <a href="#ACVotes">Advisory Committee votes</a> are described separately.
 
 <h3 id="WGAppeals">
-Appeal of a Chair's Decision</h3>
+Appeal of a Chair's Decision or of a Group Decision</h3>
```

Comment:
Maybe separate these sections out? It's a little complex being both "chairs decision" and "group decision". It may be best for one section as "Appeal of a Chair's Decision" and "Appeal of a Group Decision".

Reply:
Separating in two sections would mean duplicating most of the text, and would make it harder to explain the different between the two types of decisions. However, I do agree that the title was clumsy, so I've rephrased.